package.json

```diff
@@ -5,6 +5,9 @@
   "main": "none.js",
   "scripts": {
     "get-proposals": "node ./scripts/proposals.js",
+    "get:submodule": "git submodule update --init --depth 1",
+    "_prepare:docsy": "cd themes/docsy && npm install",
+    "prepare": "npm run get:submodule && npm run _prepare:docsy",
```

Comment:
This is taken from docsy's docs: https://www.docsy.dev/docs/get-started/other-options/#for-an-existing-site

Unfortunately, we can't add comments to package.json files...